xds/src/main/java/io/grpc/xds/internal/sds/SslContextProvider.java

```diff
@@ -93,7 +93,8 @@ public UpstreamTlsContext getUpstreamTlsContext() {
   }
 
   /** Closes this provider and releases any resources. */
-  void close() {}
+  @Override
+  public void close() {}
```

Comment:
Can this be abstract method?

Reply:
I left it as a default implementation so the subclasses don't need to define it for a "no-op" `close()`. But making abstract and forcing subclasses to define it (since closing to release resources is so important) is okay as well so I made the change